### PR TITLE
v5: Port new homepage changes

### PR DESCRIPTION
**Pull request by mdo**
Could use some refinement on the sizing and spacing IMO. Will sit on it for a couple days maybe. Feedback welcome though!

Preview: https://deploy-preview-30779--twbs-bootstrap.netlify.app/

### CODE REVIEW

**Comment by XhmikosR** [comment]
Note that we should backport any further changes from #30768.

**Comment by MartijnCuppens** [comment]
I find the gradient under this button a bit weird: 
image

Also, where is the new ~burger~ logo? 😃

**Comment by mdo** [comment]
This should be good to go. Hoping to merge tomorrow. /cc @XhmikosR

**Comment by XhmikosR** [comment]
I'll try to have a look later today, clean up the branch and backport any missing stuff.

**Comment by XhmikosR** [comment]
I backported everything from v4-dev, and saves a few more bytes from the images.

**Review by XhmikosR — CHANGES_REQUESTED** [review]
Let's move the favicon patches to a separate PR. They need more work

File: dist/css/bootstrap-grid.css.map [hunk 1]
```diff
@@ -1 +1 @@
...
....5rem !important;\n  }\n  .p-lg-5 {\n    padding: 3rem !important;\n  }\n  .px-lg-0 {\n    padding-right: 0 !important;\n    padding-left: 0 !important;\n  }\n  .px-lg-1 {\n    padding-right: 0.25rem !important;\n    padding-left: 0.25rem !important;\n  }\n  .px-lg-2 {\n    padding-right: 0.5rem !important;\n    padding-left: 0.5rem !important;\n  }\n  .px-lg-3 {\n    padding-right: 1rem !important;\n    padding-left: 1rem !important;\n  }\n  .px-lg-4 {\n    padding-right: 1.5rem !important;\n    padding-left: 1.5rem !important;\n  }\n  .px-lg-5 {\n    padding-right: 3rem !important;\n    padding-left: 3rem !important;\n  }\n  .py-lg-0 {\n    padding-top: 0 !important;\n    padding-bottom: 0 !important;\n  }\n  .py-lg-1 {\n    padding-top: 0.25rem !important;\n    padding-bottom: 0.25rem !important;\n  }\n  .py-lg-2 {\n    padding-top: 0.5rem !important;\n    padding-bottom: 0.5rem !important;\n  }\n  .py-lg-3 {\n    padding-top: 1rem !important;\n    padding-bottom: 1rem !important;\n  }\n  .py-lg-4 {\n    padding-top: 1.5rem !important;\n    padding-bottom: 1.5rem !important;\n  }\n  .py-lg-5 {\n    padding-top: 3rem !important;\n    padding-bottom: 3rem !important;\n  }\n  .pt-l
```
Comment (XhmikosR) [inline dist/css/bootstrap-grid.css.map:1]: The dist file changes shouldn't be here also

**Comment by XhmikosR** [comment]
bootstrap-social-logo.png also has an issue when I try to compress with zopli-png:

```
Compressing "bootstrap-social-logo.png"...
Optimizing bootstrap-social-logo.png
Decoding error 52: jumped past memory while inflating
```

Should be recreated so that this doesn't happen because it doesn't look right to me that this happens. We'll discuss in the other PR the other favicon related changes.

**Comment by mdo** [comment]
Favicons are now in #31042. favicon.ico still needs help I think, file size doesn't seem right but is def more manageable than my previous export from Figma.

**Comment by patrickhlauke** [comment]
the new "B" logo looks just ever so slightly off. the B cutout itself feels like it's not properly (optically) centered and feels a bit too far to the left? maybe nitpicky - and maybe it's too late - but perhaps just nudging the cutout a smidgen to the right would give a more pleasing overall look?

**Comment by patrickhlauke** [comment]
> zoomed-in version of the logo with a center line

**Comment by patrickhlauke** [comment]
did a rough tweak (using affinity designer) just as a proof of concept (but i'm sure that messed up all sorts of coordinate things).

```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg width="100%" height="100%" viewBox="0 0 118 32" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" xmlns:serif="http://www.serif.com/" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
    <path d="M47.258,0C44.966,0 43.27,2.006 43.346,4.182C43.419,6.272 43.324,8.979 42.643,11.187C41.959,13.401 40.803,14.803 38.915,14.983L38.915,17.017C40.803,17.197 41.959,18.599 42.643,20.813C43.324,23.021 43.419,25.728 43.346,27.818C43.27,29.994 44.966,32 47.259,32L70.745,32C73.037,32 74.732,29.994 74.657,27.818C74.584,25.728 74.678,23.021 75.36,20.813C76.044,18.599 77.197,17.197 79.085,17.017L79.085,14.983C77.197,14.803 76.044,13.401 75.36,11.187C74.678,8.979 74.584,6.272 74.657,4.182C74.732,2.006 73.037,0 70.745,0L47.258,0L47.258,0ZM66.269,19.698C66.269,22.694 64.035,24.511 60.327,24.511L54.014,24.511C53.64,24.511 53.333,24.203 53.333,23.83L53.333,8.17C53.333,7.797 53.64,7.489 54.014,7.489L60.29,7.489C63.383,7.489 65.412,9.164 65.412,11.736C65.412,13.541 64.047,15.157 62.307,15.44L62.307,15.534C64.675,15.794 66.269,17.433 66.269,19.698ZM59.638,9.648L56.039,9.648L56.039,14.732L59.07,14.732C61.414,14.732 62.706,13.788 62.706,12.101C62.706,10.521 61.595,9.648 59.638,9.648ZM56.039,16.749L56.039,22.352L59.771,22.352C62.21,22.352 63.503,21.373 63.503,19.533C63.503,17.692 62.174,16.749 59.614,16.749L56.039,16.749Z" style="fill:white;"/>
</svg>
```

but for comparison, squint and have a look at these two (top is the current new one, bottom is my roughly tweaked one ... subjectively, the tweaked one looks a shade more centered/balanced)

bs

**Review by XhmikosR — APPROVED** [review]

**Pull request merged**